index.html

```diff
@@ -49,5 +49,167 @@ <h2>I'am a Sotfware Developer</h2>
+            </div>
+            <button type="button" class="content-data-btn">See Project</button>
+          </div>
+        </div>
+        <div class="pro-art-paint">
+          <div class="content-data">
+            <h1>Professional Art Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="lang-btn2">
+              <div class="grid-btn1"><button type="button">Html</button></div>
+              <div class="grid-btn2"><button type="button">bootstrap</button></div>
+              <div class="grid-btn3"><button type="button">Ruby</button></div>
+            </div>
+            <button type="button" class="content-data-btn">See Project</button>
+          </div>
+        </div>
+        <div class="pro-art-paint">
+          <div class="content-data">
+            <h1>Professional Art Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="lang-btn2">
+              <div class="grid-btn1"><button type="button">Html</button></div>
+              <div class="grid-btn2"><button type="button">bootstrap</button></div>
+              <div class="grid-btn3"><button type="button">Ruby</button></div>
+            </div>
+            <button type="button" class="content-data-btn">See Project</button>
+          </div>
+        </div>
+        <div class="pro-art-paint">
+          <div class="content-data">
+            <h1>Professional Art Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="lang-btn2">
+              <div class="grid-btn1"><button type="button">Html</button></div>
+              <div class="grid-btn2"><button type="button">bootstrap</button></div>
+              <div class="grid-btn3"><button type="button">Ruby</button></div>
+            </div>
+            <button type="button" class="content-data-btn">See Project</button>
+          </div>
+        </div>
+      </section>
+    </div>
+    <footer>
+      <div class="footer-main">
+        <div class="about-me-bg">
+          <div class="about-me">
+            <div class="about-me-col">
+              <h1>About Me</h1>
+              <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              <button type="button">Get My Resume</button>
+            </div>
+          </div>
+        </div>
+        <div class="footer-nav">
+          <div class="f-line"><hr></div>
+          <div class="f-lang">
+            <div class="img-lang">
+              <img src="images/rectangle.png" alt="">
+              <h1>Language</h1>
```

Comment:
- [ ] Let's change the typography in these titles so they match the template provided.

Yours:
![image](https://user-images.githubusercontent.com/79114233/139542304-79d03b0d-f7d6-4049-8f5b-365312af2c64.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/139542310-910d6f72-c263-4c7c-8652-90b1da64b4c5.png)